pkg/proxy/service_test.go

```diff
@@ -511,12 +512,21 @@ type FakeProxier struct {
 	hostname         string
 }
 
-func newFakeProxier(ipFamily v1.IPFamily) *FakeProxier {
+func newFakeProxier(ipFamily v1.IPFamily, t time.Time) *FakeProxier {
 	return &FakeProxier{
-		serviceMap:       make(ServiceMap),
-		serviceChanges:   NewServiceChangeTracker(nil, ipFamily, nil, nil),
-		endpointsMap:     make(EndpointsMap),
-		endpointsChanges: NewEndpointChangeTracker(testHostname, nil, ipFamily, nil, false, nil),
+		serviceMap:     make(ServiceMap),
```

Comment:
did you run gofmt on this?

Reply:
my editor does when it saves the file, it is wrong indented right?

Reply:
I run it manually and it keeps it this way, :shrug:

Reply:
OK.  Not a huge deal, maybe it changed in gofmt.